spark/src/test/java/org/apache/iceberg/actions/TestExpireSnapshotsAction.java

```diff
@@ -1008,4 +1008,37 @@ public void testExpireAction() {
     Assert.assertSame("Multiple calls to expire should return the same deleted files",
         pendingDeletes, action.expire());
   }
+
+  @Test
+  public void testUseLocalIterator() {
+    table.newFastAppend()
+            .appendFile(FILE_A)
```

Comment:
nit: I think it should be 4 spaces for continued indentation. Applies to the whole test.

Reply:
Got it!